ArcBicepTemplate/FullAzureMonitorMetricsProfile.bicep

```diff
@@ -30,6 +30,18 @@ param azureMonitorWorkspaceResourceId string
   'japanwest',
   'koreacentral',
   'switzerlandnorth',
+  'brazilsoutheast',
+  'francecentral',
+  'ukwest',
+  'koreasouth',
+  'switzerlandwest',
+  'japaneast',
+  'swedencentral',
+  'canadaeast',
+  'norwaywest',
+  'southindia',
+  'australiaeast',
+  'swedensouth',
```

Comment:
i see only 12 regions (instaed of 13 new ones). is it because japanwesy was already enabled :( ?

Reply:
Yeah, exactly. That's what I noticed too. Japanwest was already enabled from last time because it was there in the results from the Microsoft.Monitor API : https://github.com/Azure/prometheus-collector/pull/609

Reply:
ok thanks .